include/zephyr/drivers/misc/devmux/devmux.h

```diff
@@ -14,6 +14,8 @@
 
 #include <stdint.h>
 
+#include <sys/types.h>
```

Comment:
This could technically be moved up a line following the coding guidelines suggested by @gmarrul

Reply:
Oops, fixed!